pydocstringformatter/run.py

```diff
@@ -98,26 +84,148 @@ def format_file(self, filename: Path) -> bool:
                         file=sys.stderr,
                     )
                 with open(filename, "w", encoding="utf-8", newline=newlines) as file:
-                    file.write(tokenize.untokenize(changed_tokens))
+                    file.write(tokenize.untokenize(formatted_tokens))
                     _utils.print_to_console(
                         f"Formatted {filename_str} 📖\n", self.config.quiet
                     )
             else:
                 sys.stdout.write(
                     _utils.generate_diff(
                         tokenize.untokenize(tokens),
-                        tokenize.untokenize(changed_tokens),
+                        tokenize.untokenize(formatted_tokens),
                         filename_str,
                     )
                 )
 
         return is_changed
 
```

Comment:
I split the following function into the file and the token handling (it also resolved a "too many branches" problem in pylint)